### ISSUE TITLE
Release v0.5.4 removes labels

### ISSUE THREAD

**Issue by keorn** [JuliaPlots/GraphRecipes.jl#125]
With the latest release now labels show up, downgrading to previous version fixes the issue. Try example from docs to reproduce: http://docs.juliaplots.org/latest/graphrecipes/examples/

**Comment by keorn** [comment]
Seems like it happens on older version too, will do some digging on what may be causing the issue.

**Comment by JackDevine** [comment]
Thanks for this. Do you have a MWE? I am having a little difficulty replicating the problem,
```julia
n = 6
g = wheel_graph(n)

add_edge!(g,1,1)

graphplot(g) |> display
g2 = barabasi_albert(n,1)
graphplot(g2,edgelabel=Dict((1,2)=>"hello"))
```
works fine for me.

**Comment by keorn** [comment]
Yeah, sorry didnt take time to narrow it down. I am plotting from LightGraphs. Will look more into it later today.

![image](https://user-images.githubusercontent.com/7750981/82138492-24fb1980-9821-11ea-8f16-58cde55e2d9e.png)

**Comment by keorn** [comment]
Ok, got the minimal version (can go more minimal if needed):
```
using GraphRecipes, Plots
graphplot([[2], [2]]; names = ["a", "b"], edgelabel=Dict((2, 2) => "c", (1, 2) => "d"))
```

On the first draw already the loop and nodes labels are missing:
![image](https://user-images.githubusercontent.com/7750981/82153897-9b703980-986a-11ea-8565-067328b178f4.png)

All subsequent draws (even other graphs) do not display any labels:
![image](https://user-images.githubusercontent.com/7750981/82153916-b8a50800-986a-11ea-9178-b4be196902ee.png)

**Comment by keorn** [comment]
The above was GR backend, just tried `plotlyjs`. This time it doesnt disappear, but the self-edge label ends up weirdly on the edge between nodes:
![image](https://user-images.githubusercontent.com/7750981/82154529-dbd1b680-986e-11ea-8781-4939e011073d.png)

**Comment by lucasvreis** [comment]
I don't know if it is related, but when I set `edgelabel = some_mat` with the `GR` backend it says:

```Warning: Attribute alias `annotation` detected in the user recipe defined for the signature (::GraphRecipes.GraphPlot). To ensure expected behavior it is recommended to use the default attribute `annotations`.```

And the edge labels are blank, like in the example above.

**Comment by congUoM** [comment]
It seems to be the problem with NaN when processing the edges. I have fixed this issue I think:
https://github.com/congUoM/GraphRecipes.jl/commit/29b061a018738cd6798272affbb6236070214238
